pkg/scheduler/generic_scheduler.go

```diff
@@ -446,6 +461,10 @@ func prioritizeNodes(
 		}
 	}
 
+	if klog.V(4).Enabled() {
```

Comment:
I think it's better to move the judgement to function `logNodeAndPluginScores`, for readers may misunderstand this function will take effect only when the level is set to 4. 
SRP maybe.

Reply:
Emm···I'm not sure, I think coders who are familiar with kubernetes will understand "klog.V(4).Enabled()" means logging level >= 4, and that is the case which the function actully work. 
Besides, in my opinion, calling function is a memory-consuming work (just in my thought, go compiler may optimize it), that's why I prefer to check the condition than calling and exiting.